README.rst

```diff
@@ -2,3 +2,33 @@ SR Comp Mixtape
 ===============
 
 A service which plays tracks throughout a competition.
+
+Configuration
+-------------
+
+A configuration of tracks and triggers to play during a competition is called
+a ``mixtape``. A mixtape is a directory containing:
+
+- A configuration file, ``playlist.yaml``, which defines trigger points,
```

Comment:
Meta: Thoughts on adding a dummy `playlist.yaml` file?

Reply:
We should

Reply:
I don't think we should consider this a blocker.

Reply:
This should probably be added as a separate PR